dev-infrastructure/docs/development-setup.md

```diff
@@ -344,7 +357,13 @@ instance_types:
 EOF
 ```
 
-3) Follow CS dev setup process:
+3) Get first-party application certificate bundle (Note that this will change when new FPA changes are introduced soon): </br>
```

Comment:
I think with the current state the ("Note that this will change when new FPA are introduced soon)" can be removed.

There also seems to be some extra html markup (</br) not needed